components/dashboard/src/repositories/detail/ConfigurationDetailPrebuilds.tsx

```diff
@@ -4,27 +4,59 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { FC } from "react";
+import { FC, useState } from "react";
 import { Configuration } from "@gitpod/public-api/lib/gitpod/v1/configuration_pb";
 import { ConfigurationSettingsField } from "./ConfigurationSettingsField";
 import { Heading3, Subheading } from "@podkit/typography/Headings";
+import { Switch } from "@podkit/switch/Switch";
+import { TextMuted } from "@podkit/typography/TextMuted";
 
 type Props = {
     configuration: Configuration;
 };
 export const ConfigurationDetailPrebuilds: FC<Props> = ({ configuration }) => {
+    // TODO: hook this up to just use configuration as state and wire up optimistic update for mutation
+    const [enabled, setEnabled] = useState(!!configuration.prebuildSettings?.enabled);
+
     return (
         <>
             <ConfigurationSettingsField>
                 <Heading3>Prebuilds</Heading3>
                 <Subheading className="max-w-lg">
                     Prebuilds reduce wait time for new workspaces. Enabling requires permissions to configure repository
                     webhooks.{" "}
-                    <a href="/docs/configure/projects/prebuilds" className="gp-link">
+                    <a
+                        href="/docs/configure/projects/prebuilds"
+                        className="gp-link"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
                         Learn more
                     </a>
                     .
                 </Subheading>
+
+                <div className="flex gap-4 mt-6">
+                    {/* TODO: wrap this in a SwitchInputField that handles the switch, label and description and htmlFor/id automatically */}
+                    <Switch checked={enabled} onCheckedChange={setEnabled} id="prebuilds-enabled" />
+                    <div className="flex flex-col">
+                        <label className="font-semibold" htmlFor="prebuilds-enabled">
+                            Prebuilds are enabled
+                        </label>
+                        <TextMuted>
+                            Enabling requires permissions to configure repository webhooks.{" "}
+                            <a
+                                href="/docs/configure/projects/prebuilds"
```

Comment:
It seems a bit hacky to use our 404 gitpod.io -> www.gitpod.io redirect. It would be great to use absolute URLs, [like we do here](https://github.com/gitpod-io/gitpod/blob/77878c34ace0a3432aadb17db583674d3e2f1f63/components/dashboard/src/user-settings/PersonalAccessTokens.tsx#L165)

```suggestion
                                href="https://www.gitpod.io/docs/configure/projects/prebuilds"
```